clang/test/CIR/CodeGen/static-vars.cpp

```diff
@@ -1,5 +1,9 @@
 // RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o %t.cir
 // RUN: FileCheck --input-file=%t.cir %s
+// RUN: %clang_cc1 -triple aarch64-none-linux-android24 -fclangir -emit-cir %s -o %t1.cir
+// RUN: FileCheck --check-prefix=CIR-AARCH64 --input-file=%t1.cir %s 
+// RUN: %clang_cc1 -triple aarch64-none-linux-android24 -fclangir -emit-llvm %s -o %t1.ll
+// RUN: FileCheck --check-prefix=LLVM-AARCH64 --input-file=%t1.ll %s 
```

Comment:
Do we need this to be AArch64-specific, or can we just reuse the existing RUN lines?

Reply:
No,  no need for comdat, I was just thinking to prepare for future target-dependent features testing in this file. But probably we should wait until that comes.